src/pages/Home/Home.page.jsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
+import { useParams } from 'react-router-dom';
 import Header from '../../components/Header';
 import VideoList from '../../components/VideoList';
+import Dialog from '../../components/Dialog';
+import VideoPlayer from '../../components/VideoPlayer';
+import { useVideo } from '../../providers/Video';
 
 function HomePage() {
-  return (
-    <div>
-      <Header />
+  const { alert } = useVideo();
+  const { videoId } = useParams();
+  let content = (
+    <>
       <h1>Welcome to the Challenge!</h1>
       <VideoList />
+    </>
+  );
+  if (videoId) {
+    content = <VideoPlayer videoId={videoId} />;
+  }
```

Comment:
you can create a page dedicated to the video player

Reply:
Cool, I'm going to try that. the instructions' page suggested conditional rendering but i can move the component to a separe page.